packages/storybook/src/react-components/button/button.stories.tsx

```diff
@@ -0,0 +1,149 @@
+import { LuxButton as Button, type LuxButtonProps } from '@lux-design-system/components-react';
+import tokens from '@lux-design-system/design-tokens/dist/index.json';
+import type { Meta, StoryObj } from '@storybook/react';
+import { type HTMLAttributes, type PropsWithChildren } from 'react';
+
+const LuxButton = (props: PropsWithChildren<LuxButtonProps> & HTMLAttributes<HTMLButtonElement>) => (
```

Comment:
Volgens mij is dit niet nodig in de React Components...
Bij de Paragraph en Heading heb ik het ook weggehaald en typescript is okay, ook als ik er een bijv een aria-* property op zet. 
ff samen naar kijken?

Reply:
Je heb gelijk, dit is niet nodig. 👍 
Ik heb het gekopieerd vanuit de Heading, ik zal het daar ook weg halen.